src/lm_buddy/jobs/_entrypoints/finetuning.py

```diff
@@ -67,35 +63,28 @@ def load_and_train(config: FinetuningJobConfig, artifact_loader: ArtifactLoader)
     trainer.train()
 
 
-def run_finetuning(
-    config: FinetuningJobConfig,
-    artifact_loader: ArtifactLoader,
-) -> FinetuningResult:
-    # Place the artifact loader in Ray object store
-    artifact_loader_ref = ray.put(artifact_loader)
-
-    # Define training function internally to capture the artifact loader ref as a closure
-    # Reference: https://docs.ray.io/en/latest/ray-core/objects.html#closure-capture-of-objects
-    def training_function(config_data: dict[str, Any]):
-        artifact_loader = ray.get(artifact_loader_ref)
-        config = FinetuningJobConfig(**config_data)
-        if is_tracking_enabled(config):
-            with wandb_init_from_config(
-                config.tracking,
-                resume=WandbResumeMode.NEVER,
-                job_type=LMBuddyJobType.FINETUNING,
-            ):
-                load_and_train(config, artifact_loader)
-        else:
-            load_and_train(config, artifact_loader)
+def training_function(config_data: dict[str, Any]):
```

Comment:
Because we are no longer injecting the `ArtifactLoader`, this class doesn't need to be an internal method anymore. Minimal changes in the functionality here except for moving this function outside of the other method.